src/csharp/Grpc.Core/ChannelCredentials.cs

```diff
@@ -125,19 +142,23 @@ public SslCredentials() : this(null, null)
         /// Creates client-side SSL credentials from
         /// a string containing PEM encoded root certificates.
         /// </summary>
-        public SslCredentials(string rootCertificates) : this(rootCertificates, null)
+        public SslCredentials(string rootCertificates, VerifyPeerCallback verifyPeerCallback = null) :
```

Comment:
adding a new default argument breaks binary compatibility. If we want to add a new argument, we need to add a new constructor.

Reply:
Removed default argument and added a new constructor.